runtime/v2/bundle.go

```diff
@@ -107,9 +106,10 @@ func NewBundle(ctx context.Context, root, state, id string, spec typeurl.Any) (b
 	}
 	if spec := spec.GetValue(); spec != nil {
 		// write the spec to the bundle
-		err = os.WriteFile(filepath.Join(b.Path, configFilename), spec, 0666)
+		specPath := filepath.Join(b.Path, oci.ConfigFilename)
+		err = os.WriteFile(specPath, spec, 0666)
 		if err != nil {
-			return nil, fmt.Errorf("failed to write %s", configFilename)
+			return nil, fmt.Errorf("failed to write %s", specPath)
```

Comment:
Might as well "fix" this while we're here, this doesn't include the error. 
```suggestion
			return nil, fmt.Errorf("failed to write %q: %w", specPath, err)
```

Reply:
Good point, in that case lets axe specPath and just wrap the error

Reply:
Probably include some context on that we failed to write the runtime spec to the bundle, instead of just "failed to write"

Reply:
Yeah it could continue to include the const that we're using perhaps (not the full path).

Although just looking at the code; it may return an plain EOF, possibly without path in the error (sigh...)

Reply:
(mostly trying to avoid overly verbose errors, as in some cases the same information is repeated 2 or 3 times, which doesn't make the error clearer)

Reply:
I did `return nil, fmt.Errorf("failed to write bundle spec: %w", err)`
Is that good enough?

Reply:
That's perfect to me